### ISSUE TITLE
update supercloud partition name

### ISSUE THREAD

**Issue by tomsilver** [Learning-and-Intelligent-Systems/predicators#1103]
from the recent downtime announcement:
>You may have noticed that we renamed the partition for the Xeon-G6 nodes to “xeon-g6-volta”. During this downtime we are removing all old partition names for these nodes, including “normal”. You do not need to specify a partition name when you request a GPU, but if you do be sure to update it to the new name.

**Comment by tomsilver** [comment]
actually I guess this is just for GPUs. @ronuchit do you know if the "normal" argument in sbatch is important, or could we just remove it?

**Comment by ronuchit** [comment]
from `man sbatch` on supercloud:
```
       -p, --partition=<partition_names>
              Request a specific partition for the resource allocation.  If not specified, the default behavior is to allow  the  slurm  controller  to  select  the
              default  partition as designated by the system administrator. If the job can use more than one partition, specify their names in a comma separate list
              and the one offering earliest initiation will be used with no regard given to the partition name ordering (although higher priority partitions will be
              considered first).  When the job is initiated, the name of the partition used will be placed first in the job record partition string.
```

so, ooooops, looks like `-p` is not priority but partition, and it didn't end up mattering cause we explicitly set partition later:
`scripts/supercloud/submit_supercloud_job.py:    cmd = ("sbatch -p normal --time=99:00:00 --partition=xeon-p8 "`
so yeah... we should remove `-p normal`, i'll open a PR

**Comment by ronuchit** [comment]
closed by #1104
